### PR TITLE
TMA-684: Data product defaults to 'default'

### PR DESCRIPTION
**Pull request by kubamahnert**

### CODE REVIEW

File: lib/gooddata/lcm/actions/collect_data_product.rb [hunk 1]
```diff
@@ -31,18 +31,13 @@ def call(params)
           domain = client.domain(domain_name) || fail("Invalid domain name specified - #{domain_name}")
 
           if params.key?(:data_product)
-            begin
-              data_product = domain.data_products(params.data_product)
-            rescue RestClient::BadRequest
-              fail 'the specified DataProduct does not exist'
-            end
+            data_product_id = params.data_product
```
Comment (yenkins) [inline lib/gooddata/lcm/actions/collect_data_product.rb]: Refers to 'params' more than self (maybe move it to another class?) (FeatureEnvy)

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/577/ : SUCCESS in 13s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/372/ : SUCCESS in 28s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/676/ : SUCCESS in 31s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/578/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/373/ : SUCCESS in 27s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/677/ : SUCCESS in 1m 47s

**Comment by panjan** [comment]
ok to merge

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-merge-compliance-check/236/ : SUCCESS in 14s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/379/ : SUCCESS in 1m 45s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/683/ : SUCCESS in 2m 00s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1135/ : SUCCESS in 48m 07s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/893/ : SUCCESS in 28m 54s

**Pull request merged**